cognite/neat/_rules/analysis/_base.py

```diff
@@ -209,6 +215,13 @@ def logical_uri_by_property_by_view(
             for view, properties in properties_by_view.items()
         }
 
+    @property
+    def class_by_neat_id(self) -> dict[URIRef, InformationClass]:
```

Comment:
Private method? I think we should not expose `neat_id` outside rules including this class.